src/statement.d

```diff
@@ -1324,38 +1329,38 @@ extern (C++) class CompoundStatement : Statement
         return statements;
     }
 
-    override final ReturnStatement isReturnStatement()
+    override final inout(ReturnStatement) isReturnStatement() inout nothrow pure
```

Comment:
Fine. But I'll open another PR for that. It is outside of the scope for this one.